src/main/java/net/dries007/tfc/common/blocks/crop/DeadClimbingCropBlock.java

```diff
@@ -6,21 +6,28 @@
 
 package net.dries007.tfc.common.blocks.crop;
 
+import java.util.function.Supplier;
 import net.minecraft.world.level.block.Block;
 import net.minecraft.world.level.block.state.BlockState;
 import net.minecraft.world.level.block.state.StateDefinition;
 import net.minecraft.world.level.block.state.properties.BooleanProperty;
 
 import net.dries007.tfc.common.blocks.ExtendedProperties;
 import net.dries007.tfc.common.blocks.TFCBlockStateProperties;
+import net.dries007.tfc.util.climate.ClimateRange;
 
 public class DeadClimbingCropBlock extends DeadDoubleCropBlock
 {
     public static final BooleanProperty STICK = TFCBlockStateProperties.STICK;
 
     public DeadClimbingCropBlock(ExtendedProperties properties, Crop crop)
     {
-        super(properties, crop);
+        this(properties, crop.getClimateRange());
+    }
+
+    public DeadClimbingCropBlock(ExtendedProperties properties, Supplier<ClimateRange> range)
+    {
+        super(properties, range);
```

Comment:
Do dead climbing crops actually require a climate range? It makes more sense to me intuitively to pass in `() -> ClimateRange.NOOP` to the super constructor here, as it's explicitly intended to be unused. (See `DeadBerryBushBlock`)

Reply:
you need it for the hoe overlay so you can see why it didn't grow

Reply:
Makes sense.